content/target_mode.md

```diff
@@ -0,0 +1,406 @@
+
+```ruby
+provides :<RESOURCE_NAME>, target_mode: true
+...
+```
+
+See the [Custom Resources documentation]({{< relref "custom_resources" >}}) for more detailed documentation about creating custom resources.
+
+#### Example
+
+The following custom resource example checks for and creates a new directory and runs in Target Mode:
+
+```ruby
+provides :example_directory, target_mode: true
+unified_mode true
+
+property: directory, String
+
+load_current_value do |new_resource|
+  dir = new_resource.directory
+  parsed = dir.match(%r{([^/]+$)})
+  path = ''
+  if parsed
+    path = dir[0..(dir.length - parsed[1].length - 1)]
+    dir = parsed[1]
+  end
+
+  tmp = __transport_connection.run_command( sprintf('ls -l %s | grep %s || echo -n', path, dir) )
+
+  if tmp.match(Regexp.new(dir))
+    directory new_resource.directory
+  end
+end
+
+action :create do
+  converge_if_changed do
+    __transport_connection.run_command( sprintf('mkdir %s', new_resource.directory) )
+  end
+end
+```
+
+## Run Target Mode
+
+Run the `chef-client` executable using `-t` or `--target` to target a specific node. For example:
+
+```sh
+chef-client -t <TARGET_NAME>
+```
+
+Replace `<TARGET_NAME>` with the name of the host as defined in the credentials file. For example, `HOST-1` in the [credential file example](#examples).
+
+To execute a specific Cookbook in Target Mode, run:
+
+```sh
+chef-client -t <TARGET_NAME> <PATH/TO/COOKBOOK/COOKBOOK_NAME>
+```
+
+Replace the following:
+
+- `<TARGET_NAME>` with the name of the host as defined in the credentials file.
+- `<PATH/TO/COOKBOOK/COOKBOOK_NAME>` with the path to the Cookbook on your system. For example, `/chef-repo/cookbooks/example_cookbook.rb`
+
+### Target Mode in Local Mode
+
+You can run Target Mode in [Local Mode]({{< relref "/ctl_chef_client#run-in-local-mode" >}}).
+Local Mode runs chef-zero locally as a lightweight instance of Chef Infra Server to execute a Client run on target nodes.
+
+Use `-z` and `-t` to run Target Mode in Local Mode:
+
+```sh
+chef-client -z -t <TARGET_NAME>
+```
+
+Replace `<TARGET_NAME>` with the name of the host as defined in the credentials file. For example, `HOST-1` in the [credential file example](#examples).
+
+## Run Target Mode with Chef Automate or Chef Infra Server
+
+You can configure Chef Automate or Chef Infra Server to run Target Mode on a regular schedule.
+
+Target Mode doesn't have a way to schedule Chef Infra Client runs on a node, but you can create a cron file that executes Target Mode on a regular schedule.
+
+For example, this create a cron file that executes Target Mode every thirty minutes:
+
+```ruby
+cat > /etc/cron.d/nodename.cron <<EOF
+PATH="/opt/chefdk/bin:/opt/chefdk/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+GEM_ROOT="/opt/chefdk/embedded/lib/ruby/gems/2.6.0"
+
+0,30 * * * * * chef-runner /usr/bin/chef-client -t <TARGET_NAME>
```

Comment:
As per my knowledge there is  missing peace of information here. 
1. Where this cron file should be?
2. Second when chef-client without --local-mode i.e -z option will check for the client.pem file and client.rb file. That peace of information is missing here. Unless you have client.pem and client.rb file on the host the chef-client run will fail if not running in local mode.

Reply:
@Vasu1105 Given target mode, I would assume the host is the only one with chef-client, and the usual requirements for chef-client setup would apply. It might be worth calling that out specifically still.